mqtt/tls-connect-to-broker.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+title: Connect to an MQTT broker with TLS encryption
+slug:
+  - label: mqtt
+    url: /#mqtt
+  - secure connect
+---
+
+### Problem
+
+You want to have an encrypted connection to your Mosquitto MQTT broker.
+
+### Solution
+
+Create a valid set of certificates and keys for the broker to use.
+
+Change the configuration of the broker to start a TLS encrypted port (`mqtts`) using the above.
+
+Alter the <code class="node">MQTT Config</code> node, changing the "Server" name to start with `mqtts://`.
+
+#### Example
+
+**Valid Certificate Creation**
+
+THe easiest approach for this is to use [Let's Encrypt](). This is beyond the scopy of this article but there are plenty of examples and tutorials available on the Internet. For this to work successfully, you also need to be able to use a registered domain name on your internal network because you cannot use Let's Encrypt with IP addresses or non-public domain names.
```

Comment:
scopy ?

Reply:
Of course "scopy", why not! 🥇 
Sorry about the missing link.